french/src/recommended-wallets.md

```diff
@@ -2,7 +2,8 @@
 
 Les portefeuilles suivants sont ceux qui ont montré de meilleures performances lorsqu'ils ont été testés avec le bot :
 
+- [Wallet of Satoshi](https://www.walletofsatoshi.com/)
```

Comment:
It hasn't really performed that well with the bot, for example if you cancel a Sats sale it takes a few hours to update your wallet balance.

Reply:
ok thank you for the notice. but apart for cancellation delay, it is very speedy for normal transactions and also widely used. hope they will fix that in future versions...